apps/dashboard/app/views/projects/import.html.erb

```diff
@@ -53,3 +52,31 @@
   </p>
 </div>
 <% end %>
+
+<div class='card'>
+  <div class='card-group'>
+    <div class='card'>
+      <div class='card-body'>
+        <strong>
+          <%= label :directory, I18n.t('dashboard.jobs_select_directory_placeholder') %>
+        </strong>
+        <div class="d-flex flex-wrap justify-content-start">
+          <% @projects.each do |project| %>
```

Comment:
This fails for me, claiming that `@projects` is nil. Don't see it in the controller...

Reply:
My bad, I didn't update the controller code.